src/administration/web/configure-environment.md

```diff
@@ -1,14 +1,22 @@
 # Environment configuration
 
-You can access the configuration page of an environment by clicking the configure link under the environment name.
+You can access an environment's settings by selecting that environment from the `Select Environments` pull-down menu at the top of the page or by clicking that environment within the Environments graphic on the right side. Click the `Settings` tab at the top of the screen.
 
-![image](/images/ui-conf-environment.png)
+## General
 
-## Settings
+The `General` screen allows you to extend the behavior of a specific environment.
 
-The `Settings` screen allows you to extend the behavior of a specific environment.
+![env gen settings](/images/env_settings.png)
 
-![Configure Platform.sh environment settings](/images/ui-conf-environment-settings.png)
+### Environment name
+
+The first setting allows you to modify the name of the environment, and view its parent environment.
+
+### Status
+
+From the `Status` tab, an active environment can be de-activated. 
+
+![env status](/images/env_status.png)
 
 The *Delete* action will destroy all services running on this environment (PHP, MySQL, volumes, etc.) so that only the Git branch remains. You can reactivate the environment later if needed.
```

Comment:
Does it now say Delete or Deactivate?  Docs should be consistent with whichever it is.